<?xml version="1.0"?>
<launch>
    <arg name="bagPath" />
    <arg name="videoPath" />
    <arg name="speed" default="1" />
    <arg name="videoConfig" default="$( find navigation_test_analysis )/videoConfigExample.yaml" />
    <arg name="keepalive" default="true" />
    <arg name="repository" default="fromBagfile" />
    <node name="analyse_remaining_bag_files" pkg="navigation_test_analysis" type="workerPool.py" output="screen" required="true">
        <param name="~bagPath"     value="$( arg bagPath )" />
        <param name="~videoPath"   value="$( arg videoPath )" />
        <param name="~videoConfig" value="$( arg videoConfig )" />
        <param name="~speed"       value="$( arg speed )" />
        <param name="~keepalive"   value="$( arg keepalive )" />
        <param name="~repository"  value="$( arg repository )" />
    </node>
</launch>
